pages/health-beta/index.md

```diff
@@ -1,11 +0,0 @@
----
-title: Your VA.gov Account
-layout: page-react.html
-entryname: health-beta
```

Comment:
There doesn't seem to be an associated manifest for this page. I can't find an `entryName` of `health-beta`.